vhdbuilder/packer/vhd-image-builder-mariner2-arm64-gen2.json

```diff
@@ -0,0 +1,298 @@
+{
+    "variables": {
+      "client_id": "{{env `AZURE_CLIENT_ID`}}",
+      "client_secret": "{{env `AZURE_CLIENT_SECRET`}}",
+      "tenant_id": "{{env `AZURE_TENANT_ID`}}",
+      "subscription_id": "{{env `AZURE_SUBSCRIPTION_ID`}}",
+      "location": "{{env `AZURE_LOCATION`}}",
+      "vm_size": "{{env `AZURE_VM_SIZE`}}",
+      "build_number": "{{env `BUILD_NUMBER`}}",
+      "build_id": "{{env `BUILD_ID`}}",
+      "commit": "{{env `GIT_VERSION`}}",
+      "feature_flags": "{{env `FEATURE_FLAGS`}}",
+      "os_version": "{{env `OS_VERSION`}}",
+      "hyperv_generation": "{{env `HYPERV_GENERATION`}}",
+      "container_runtime": "{{env `CONTAINER_RUNTIME`}}",
+      "teleportd_plugin_download_url": "{{env `TELEPORTD_PLUGIN_DOWNLOAD_URL`}}",
+      "arm64_os_disk_snapshot_name": "{{env `${ARM64_OS_DISK_SNAPSHOT_NAME`}}",
+      "gen2_captured_sig_version": "{{env `${GEN2_CAPTURED_SIG_VERSION`}}",
+      "enable_fips": "{{env `ENABLE_FIPS`}}",
+      "img_publisher": "{{env `IMG_PUBLISHER`}}",
+      "img_offer": "{{env `IMG_OFFER`}}",
+      "img_sku": "{{env `IMG_SKU`}}"
+    },
+    "builders": [
+      {
+        "type": "azure-arm",
+        "client_id": "{{user `client_id`}}",
+        "client_secret": "{{user `client_secret`}}",
+        "tenant_id": "{{user `tenant_id`}}",
+        "subscription_id": "{{user `subscription_id`}}",
+        "os_type": "Linux",
+        "os_disk_size_gb": 30,
+        "image_publisher": "{{user `img_publisher`}}",
+        "image_offer": "{{user `img_offer`}}",
+        "image_sku": "{{user `img_sku`}}",
+        "image_version": "latest",
+        "azure_tags": {
+          "SkipLinuxAzSecPack": "true",
+          "os": "Linux",
+          "now": "{{user `create_time`}}",
+          "createdBy": "aks-vhd-pipeline"
+        },
+        "location": "{{user `location`}}",
+        "vm_size": "{{user `vm_size`}}",
+        "custom_data_file": "vhdbuilder/packer/sha1rsa_custom_data",
+        "managed_image_os_disk_snapshot_name": "{{user `arm64_os_disk_snapshot_name`}}",
+        "managed_image_resource_group_name": "{{user `resource_group_name`}}",
+        "managed_image_name": "{{user `sig_image_name`}}-{{user `gen2_captured_sig_version`}}",
+        "shared_image_gallery_destination": {
```

Comment:
Please refer to my recent AB PR https://github.com/Azure/AgentBaker/pull/1956 for corresponding update to this file.

Reply:
Similarly addressed!